requirements/base.in

```diff
@@ -1,7 +1,7 @@
 # Base requirements - for all installations
 bagit==1.7.0
-boto3==1.9.174
-botocore==1.12.253
+boto3>=1.17.88,<1.18.0
+botocore
```

Comment:
`botocore` doesn't need to be pinned anymore?

Reply:
I wasn't sure to what version, it's boto3 that have a hard constrain. What would you do?

Reply:
Leave `botocore` out of the `.in` file and let `pip-tools` to deal with it in the `.txt` files?

Reply:
I can do that, thanks!